manticore/manticore.py

```diff
@@ -404,10 +370,11 @@ def enqueue(self, state):
     ###########################################################################
     # Workers                                                                 #
     ###########################################################################
-    def _start_workers(self, num_processes, profiling=False):
+    def _start_workers(self, num_processes, profiling=False, verbose=True):
         assert num_processes > 0, "Must have more than 0 worker processes"
 
-        logger.info("Starting %d processes.", num_processes)
```

Comment:
I was questioning whether the "# processes" line even belongs in regular manticore output at the info level. i think we should demote this to a debug line. then we can remove the hack to suppress this line just for evm. i think we can remove that line with a general "beginning analysis" output at the info level